src/FixedDataTableCell.js

```diff
@@ -244,7 +249,15 @@ var FixedDataTableCell = createReactClass({
         <div
           className={cx('fixedDataTableCellLayout/columnResizerContainer')}
           style={columnResizerStyle}
-          onMouseDown={this._onColumnResizerMouseDown}>
+          /**
+           * Stop the event from being propagated when touching the handle.
+           * This prevents the rows from moving around when we resize the headers.
+           */
+          onTouchStart={e => e.stopPropagation()}
```

Comment:
how does the div handle having 2 onTouchStart props?  Shouldn't we combine them?

Reply:
Not sure how I missed the 2 onTouchStart. I am going to combine them as suggested and stopPropagation only if touch is enabled since _onColumnResizerMouseDown is called onMouseDown as well.